Quadriga/src/main/java/edu/asu/spring/quadriga/accesschecks/IProjectSecurityChecker.java

```diff
@@ -5,31 +5,23 @@
 
 public interface IProjectSecurityChecker {
 
-	public abstract boolean checkProjectAccess(String userName,String projectId)
-			throws QuadrigaStorageException;
+    public abstract boolean checkProjectAccess(String userName, String projectId) throws QuadrigaStorageException;
 
-	public abstract boolean isProjectOwner(String userName,String projectId)
-			throws QuadrigaStorageException;
+    public abstract boolean isProjectOwner(String userName, String projectId) throws QuadrigaStorageException;
 
-	public abstract boolean checkQudrigaAdmin(String userName)
-			throws QuadrigaStorageException;
+    public abstract boolean checkQudrigaAdmin(String userName) throws QuadrigaStorageException;
 
-	public abstract boolean isUserCollaboratorOnProject(String userName, String projectId,
-			String collaboratorRole) throws QuadrigaStorageException;
+    public abstract boolean isUserCollaboratorOnProject(String userName, String projectId, String collaboratorRole)
+            throws QuadrigaStorageException;
 
-	public abstract boolean isEditor(String userName, String projectId)
-			throws QuadrigaStorageException;
```

Comment:
This method is removed and 'isCollaborator" method is used in where this method was used.